packages/updatePrescriptionStatus/tests/utils/testUtils.ts

```diff
@@ -38,6 +38,7 @@ const TASK_VALUES = [
     id: TASK_ID_0,
     status: "completed",
     businessStatus: "Dispatched",
+    repeatNo: 1,
```

Comment:
There's something a bit odd going on in this file. This line can be removed without the tests failing.

Reply:
The optional field `repeatNo` has been removed.